tsconfig.json

```diff
@@ -7,7 +7,9 @@
     "noImplicitOverride": true,
     "noUncheckedIndexedAccess": true,
     "allowJs": true,
-    "noEmit": true
+    "noEmit": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Could you explain why you toggled these both on? 🙂

Reply:
esModuleInterop was a mistake
allowSyntheticDefaultImports was required for @poppinss/dev-utils but I removed it because it is not used anymore